src/components/header/index.js

```diff
@@ -1,102 +1,114 @@
-                                title='team logo'
-                                alt='team logo'
-                            />
-                        </div>
-                        <div className={styles.selectContainer}>
-                            <select
-                                value={selectedTeamId}
-                                onChange={this.onSelectChange}
-                                style={{ backgroundColor: `${selectedTeamColor}`}}
-                            >
-                                {teams.map((team) => {
-                                    return <option key={team.teamId} value={team.teamId}>{team.ttsName}</option>;
-                                })}
-                            </select>
-                        </div>
-                    </div>
-                    <div
-                        style={{ borderColor: `${selectedTeamColor || COLORS.LIGHT_GREY} transparent transparent transparent` }}
-                        className={styles.borderTriangle}
-                    />
-                </div>
-
-                <NavMenu selectedTeamColor={selectedTeamColor} />
-
-                <div className={styles.user}>
-                    <button>
-                        <img src={bell} alt='notification bell'></img>
-                    </button>
-                    <button>
-                        <img src={settingsIcon} alt='settings icon'></img>
-                    </button>
-                    <button>
-                        <img src={kobe} alt='user avatar' height='25' className={styles.userPic}></img>
-                        <div>Kobe Bryant</div>
-                        <i className={styles.dropDownUser}></i>
-                    </button>
-                </div>
-            </div>
-        );
-    };
+  componentDidMount() {
+    this.props.getTeams();
+  };
+
+  onSelectChange = (e) => {
+    this.props.getSelectedTeam(e);
+  };
+
+  CustomValue = ( props ) => {
+    const { data: { primaryColor, ttsName, tricode }, innerRef, innerProps } = props;
+
+    return (
+      <div ref={innerRef} {...innerProps} style={{ backgroundColor: `${primaryColor}`}} className={styles.optionContainer}>
+        <div className={styles.optionImgContainer}>
+          <img
+            src={`https://cdn.nba.net/assets/logos/teams/secondary/web/${tricode}.svg`}
+            title='team logo'
+            alt='team logo'
+            className={styles.optionImage}
+          />
+        </div>
+        <div className={styles.optionTitle}>{ttsName}</div>
+      </div>
+    );
+  };
+
+  CustomOption = ( props ) => {
+    const { data: { primaryColor, ttsName, tricode, teamId }, innerRef, innerProps } = props;
+    const { selectedTeam } = this.props;
+    let isSelected = selectedTeam.teamId === teamId;
+
+    return (
+      <div ref={innerRef} {...innerProps} style={{ backgroundColor: `${primaryColor}` }} className={classnames(styles.optionContainer, isSelected && styles.displayNone)}>
```

Comment:
instead of setting displayNone on isSelected, only show the entire option is isSelected is false:

```
    return !isSelected &&
      <div ref={innerRef} {...innerProps} style={{ backgroundColor: `${primaryColor}` }} className={styles.optionContainer}>
        <div className={styles.optionImgContainer}>
          <img
            src={`https://cdn.nba.net/assets/logos/teams/secondary/web/${tricode}.svg`}
            title='team logo'
            alt='team logo'
            className={styles.optionImage}
          />
        </div>
        <div className={styles.optionTitle}>{ttsName}</div>
      </div>
    ;
  };
```